lib/msf/core/exploit/remote/http/nagios_xi/rce_check.rb

```diff
@@ -0,0 +1,76 @@
+# -*- coding: binary -*-
+
+module Msf::Exploit::Remote::HTTP::NagiosXi::RceCheck
+  # Uses the Nagios XI version to check which CVEs and related exploit modules the target is vulnerable to, if any
+  #
+  # @param version [Gem::Version] Nagios XI version
+  # @return [Hash], Hash mapping CVE numbers to exploit module names if the target is vulnerable, empty hash otherwise
+   def nagios_xi_rce_check(version)
+    matching_exploits = {}
+    
+    # known exploits that affect versions prior to the one in the hash key
+    nagios_rce_version_prior = {}
+
+    nagios_rce_version_prior.each do |fixed_version, info|
+      if version < Gem::Version.new(fixed_version)
+        matching_exploits = add_cve_module_to_hash(matching_exploits,info)
```

Comment:
```suggestion
        matching_exploits = add_cve_module_to_hash(matching_exploits, info)
```

Reply:
there were some more instances of this, changed them all